dataprofiler/profilers/profile_builder.py

```diff
@@ -21,6 +21,7 @@
 
 from .. import data_readers, dp_logging
 from ..labelers.data_labelers import DataLabeler
+from ..labelers.base_data_labeler import BaseDataLabeler
```

Comment:
adding this import so the code can set the `replacement_type` for when `_restore_data_labelers` is called on a profile object